tests/WebAuthn_forge/src/FCL_Webauthn.sol

```diff
@@ -34,8 +34,8 @@ library FCL_WebAuthn {
         bytes calldata clientData,
         bytes32 clientChallenge,
         uint256 clientChallengeDataOffset,
-        uint256[2] calldata rs
-    ) internal returns (bytes32 result) {
+        uint256[2] calldata // rs
```

Comment:
By making this unused argument nameless, it's explicit we don't use it. In addition that resolves the compiler warning associated to it.